teuthology/misc.py

```diff
@@ -1057,7 +1057,7 @@ def get_valgrind_args(testdir, name, preamble, v):
         return preamble
     if not isinstance(v, list):
         v = [v]
-    val_path = '/var/log/ceph/valgrind'.format(tdir=testdir)
+    val_path = '/var/log/ceph/valgrind'
```

Comment:
Is this related to the title of the PR? It looks for me it is not. If so, should be in separate commit.

Reply:
It is. Without that change, `mypy` will complain that this code is not correct.